pyEpiabm/pyEpiabm/intervention/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .place_closure import PlaceClosure
 from .household_quarantine import HouseholdQuarantine
 from .social_distancing import SocialDistancing
+from .testing import Testing
```

Comment:
This could be confused with software testing - can we rename to `DiseaseTesting`?